### DISCUSSION TITLE
Sonoff Dual R3 keeps rebooting using Termite terminal

### DISCUSSION THREAD

**Discussion by HaViGit** [arendst/Tasmota#21147] [Q&A] [answered]
After I flashed tasmota32-NL.bin without error messages on my new Sonoff Dual R3, I am unable to make settings via the console. I use programs such as Termite and that has always worked well with other Sonoff products. With Termite I can connect to the Dual R3 but the system keeps rebooting with the output below.

Does anyone know what I'm doing wrong?

Termite output:

_rst:0x3 (SW_RESET),boot:0x13 (SPI_FAST_FLASH_BOOT)
configsip: 0, SPIWP:0xee
clk_drv:0x00,q_drv:0x00,d_drv:0x00,cs0_drv:0x00,hd_drv:0x00,wp_drv:0x00
mode:DIO, clock div:2
load:0x3fff0018,len:4
load:0x3fff001c,len:1044
load:0x40078000,len:8896
load:0x40080400,len:5828
entry 0x400806ac
ets Jul 29 2019 12:21:46_

ESPHome-Flasher output after succesful flash:

<img width="548" alt="flasher" src="https://github.com/arendst/Tasmota/assets/46343309/de7ead24-7c60-4500-aaca-5811baca8fb0">

**Comment by sfromis** [comment]
Better use the *recommended* tools instead of just digging up whatever else....

What you posted is a highly **un**successful flash, using the wrong binary.

Using the Tasmota web installer is recommended to have it automatically do the right thing.... https://tasmota.github.io/install/

And that is that initial flashing needs to be using a `.factory` variant of the binary.

**Comment by HaViGit** [comment]
The file tasmota32-NL that I used was official and I downloaded it from http://ota.tasmota.com/tasmota32/release. It also didn't work when I flashed with https://tasmota.github.io/install.

I also tried the (official) file tasmota32-NL.**factory**.bin that I installed via https://tasmota.github.io/install. That gives the same problem with output as below. The problem also occurs with a file from the dropdown (Tasmota32 Bluetooth English).

Scrolling terminal output:

rst:0x1 (POWERON_RESET),boot:0x13 (SPI_FAST_FLASH_BOOT)
 configsip: 0, SPIWP:0xee
 clk_drv:0x00,q_drv:0x00,d_drv:0x00,cs0_drv:0x00,hd_drv:0x00,wp_drv:0x00
 mode:DIO, clock div:2
 load:0x3fff0030,len:184
 load:0x40078000,len:12732
 ho 0 tail 12 room 4
 load:0x40080400,len:2908
 entry 0x400805c4
 
 00:00:00.003-215/50 HDW: ESP32-D0WD-V3 v3.1
 00:00:00.023-213/50 UFS: FlashFS mounted with 308 kB free
 00:00:00.036 CFG: Loaded from File, Aantal 2
 ets Jul 29 2019 12:21:46
 
 rst:0xc (SW_CPU_RESET),boot:0x13 (SPI_FAST_FLASH_BOOT)
 configsip: 0, SPIWP:0xee
 clk_drv:0x00,q_drv:0x00,d_drv:0x00,cs0_drv:0x00,hd_drv:0x00,wp_drv:0x00
 mode:DIO, clock div:2
 load:0x3fff0030,len:184
 load:0x40078000,len:12732
 ho 0 tail 12 room 4
 load:0x40080400,len:2908
 entry 0x400805c4
 
 00:00:00.003 HDW: ESP32-D0WD-V3 v3.1
 00:00:00.022 UFS: FlashFS mounted with 312 kB free
 00:00:00.025 TFS: File '.settings' not found
 00:00:00.026 CFG: Loaded from NVS, Aantal 2
 00:00:00.027 FRC: Some settings have been reset (2)
 00:00:00.459 BRY: Berry initialized, RAM used 5180 bytes
 00:00:00.477 Project tasmota - Tasmota Versie 13.4.0(tasmota32)-2_0_14(2024-02-14T16:23:06)
 00:00:00.883 WIF: WifiManager 3 minuten actief
 ets Jul 29 2019 12:21:46

**Reply by sfromis** [reply]
No, that is not even close to being "the same" problem with output, it is after a successful flash and boot where Tasmota is waiting for you to do initial setup via Wifi. However, do note that you cannot expect this to work via power from the USB/serial adapter; disconnect this and use mains instead.

**Comment by HaViGit** [comment]
This is the PCB of my Dual R3. Because it seems to be an older hardware version I've also flashed older Tasmota versions (8.5 and 9.5) with the same results :-(

![IMG_20240412_084042578_HDR~2](https://github.com/arendst/Tasmota/assets/46343309/730c05dd-ca65-40e6-a899-303a4ce90fe0)

**Comment by HaViGit** [comment]
Thanks for your comment. You say that I have to supply it with 230 volts and that is certainly the reason, but how can I reach the Dual R3 via WiFi in this way? I think I don't have a connection if I can't first enter the WiFi credentials via the console, or am I missing something?

**Reply by arendst** [reply]
Yep.

Tasmota will start a WiFi Access Point to allow for configuring the WiFi credentials. Just use your phone and look for it.

**Reply by sfromis** [reply, marked as answer]
And when you have connected to the AP from Tasmota, you can select your network AP, and then enter the password.

**Comment by HaViGit** [comment]
Accessing as AP was successful after I flashed the .factory bin-file as advised. Of course, I should have known that myself... 

Anyway, both thank you for your kind help, I can now continue using the Dual R2 :-)
